homeassistant/components/totalconnect/strings.json

```diff
@@ -80,6 +80,42 @@
   "exceptions": {
     "invalid_pin": {
       "message": "Incorrect code entered"
+    },
+    "disarm_failed": {
+      "message": "TotalConnect failed to disarm {device}"
+    },
+    "disarm_invalid_code": {
+      "message": "TotalConnect usercode is invalid. Did not disarm"
+    },
+    "arm_home_failed": {
+      "message": "TotalConnect failed to arm home {device}"
+    },
+    "arm_home_invalid_code": {
+      "message": "TotalConnect usercode is invalid. Did not arm home"
+    },
+    "arm_away_failed": {
+      "message": "TotalConnect failed to arm away {device}"
+    },
+    "arm_away_invalid_code": {
+      "message": "TotalConnect usercode is invalid. Did not arm away"
+    },
```

Comment:
Personally think we should not have to add the integration name before every message